src/args.rs

```diff
@@ -0,0 +1,37 @@
+use clap::{Args, Parser, Subcommand};
+
+#[derive(Debug, Parser)]
+#[clap(author, version, about)]
+pub struct PokerArgs {
+    #[command(subcommand)]
+    /// Draw hands or get statistics
+    pub command: Option<ArgsCommands>,
+
+    #[arg(long = "sorted-deck")]
+    /// To print sorted deck
+    pub sorted_deck: bool,
+}
+
+#[derive(Debug, Subcommand)]
```

Comment:
What does `Subcommand` do here?

Reply:
Subcommand allows us to restrict the program to take only one command at a time. Like here we can use only either draw-hands or statistics or sorted deck. Only one of them is allowed to be executed unlike the arguments which can be used as many as we want all together

Reply:
Neat -- thanks for explaining!